system/security/LdapSecurity/ldapconnection.cpp

```diff
@@ -1935,6 +1937,10 @@ class CLdapClient : implements ILdapClient, public CInterface
         if(infotype && stricmp(infotype, "sudoers") == 0)
         {
             CLdapSecUser* ldapuser = dynamic_cast<CLdapSecUser*>(&user);
+            if (ldapuser == nullptr)
+            {
+                throw MakeStringException(-1, "Unable to cast input user variable");
```

Comment:
Might be more informative if you provide the user. Also, not sure if it will be obvious in context, but should we denote that it couldn't cast to an LDAP user?